x-pack/plugins/fleet/server/services/output.ts

```diff
@@ -194,18 +194,18 @@ async function validateTypeChanges(
 async function updateFleetServerPoliciesDataOutputId(
   soClient: SavedObjectsClientContract,
   esClient: ElasticsearchClient,
-  data: Partial<Output>,
+  data: Nullable<Partial<OutputSOAttributes>>,
```

Comment:
The type here was not correct, it should have been `OutputSOAttributes` already. This type doesn't have `id` so I removed the check below that it's already not working.

Reply:
nit why the `Nullable` is needed here?